EIPS/eip-2070.md

```diff
@@ -24,36 +24,24 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Berl
 
 ### Accepted EIPs
 
-- TBD
-
-### Tentatively Accepted EIPs
-
-- [EIP-663](https://eips.ethereum.org/EIPS/eip-663): Unlimited SWAP and DUP instructions
-- [EIP-1057](https://eips.ethereum.org/EIPS/eip-1057): ProgPoW, a Programmatic
-  Proof-of-Work
-  - There is a
-    [pending audit](https://medium.com/ethereum-cat-herders/progpow-audit-goals-expectations-75bb902a1f01),
-    above and beyond standard security considerations, that should be evaluated
-    prior to inclusion.
-- [EIP-1380](https://eips.ethereum.org/EIPS/eip-1380): Reduced gas cost for call to self
-- [EIP-1702](https://eips.ethereum.org/EIPS/eip-1702): Generalized account versioning scheme
-- [EIP-1962](https://eips.ethereum.org/EIPS/eip-1962): EC arithmetic and pairings with runtime definitions
-  - replaces EIP-1829
-- [EIP-1985](https://eips.ethereum.org/EIPS/eip-1985): Sane limits for certain EVM parameters
-- [EIP-2045](https://eips.ethereum.org/EIPS/eip-2045): Particle gas costs for EVM opcodes
-- [EIP-2046](https://eips.ethereum.org/EIPS/eip-2046): Reduced gas cost for static calls made to precompiles
+- [EIP 2315](https://github.com/ethereum/EIPs/pull/2576): Simple Subroutines for the EVM 
```

Comment:
Only realised after approving this points to something other than 2315.

Also that this PR removed the dashes from the "EIP-2315".

Reply:
Fixed in #2650

Reply:
Thanks, @axic